packages/docs/src/styles/_syntax.scss

```diff
@@ -0,0 +1,85 @@
+@import "variables";
+
+$light-scopes: (
+  "attribute-name": $orange3,
+  "background": $white,
+  "comment": $gray2,
+  "constant": $turquoise2,
+  "function": $blue2,
+  "id": $gold2,
+  "keyword": $violet3,
+  "numeric": $rose2,
+  "operator": $violet2,
+  "pseudo-class": $rose2,
+  "punctuation": $dark-gray3,
+  "string": $lime1,
+  "tag": $forest2,
+  "text": $dark-gray1,
+  "type": $gold2,
+  "variable": $turquoise2,
+);
+
+$dark-scopes: (
+  "attribute-name": $orange4,
+  "background": $dark-gray2,
+  "comment": $gray2,
+  "constant": $turquoise5,
+  "function": $blue4,
+  "id": $gold5,
+  "keyword": $violet4,
+  "numeric": $rose4,
+  "operator": $violet5,
+  "pseudo-class": $rose3,
+  "punctuation": $light-gray5,
+  "string": $lime4,
+  "tag": $forest3,
+  "text": $gray5,
+  "type": $gold4,
+  "variable": $turquoise3,
+);
+
+
+@mixin syntax($scopes) {
```

Comment:
minor: calling this argument "scopes" is a little confusing; I would just call it "colors" or something